src/presenter/trip-event-presenter.js

```diff
@@ -1,67 +1,60 @@
-import { render, replace } from '../framework/render.js';
-import ListEvent from '../view/event-list-view.js';
-import EventsItemView from '../view/events-item-view.js';
-import EventEdit from '../view/event-edit.js';
+import { render } from '../framework/render.js';
 import ListEmpty from '../view/list-empty.js';
-export default class EventPresenter {
+import PointPresenter from './point-presenter.js';
+import EventsListView from '../view/events-list-view.js';
 
-  #eventComponent = new ListEvent();
+export default class EventPresenter {
+  #eventsListView = new EventsListView();
   #points = null;
   #listContainer = null;
   #point = null;
   #offer = null;
   #destination = null;
 
+  #pointPresenter = null;
+
   constructor({listContainer, pointsModel, offersModel, destinationsModel}){
```

Comment:
рекомендую называть перменные по названиям классам, иначе неочевидно это массив или модель или объект или что конкретно в points/offers/desctination
```suggestion
 constructor({listContainer, pointsModel, offersModel, destinationsModel}) {
    this.#listContainer = listContainer;
    this.#pointsModel = pointsModel;
    this.#offersModel = offersModel;
    this.#destinationsModel = destinationsModel;
    this.#points = new Map();
  }
```